models/classes/class.ItemCompiler.php

```diff
@@ -27,16 +27,8 @@
  * @package taoItems
  
  */
-class taoItems_models_classes_ItemCompiler extends tao_models_classes_Compiler
+abstract class taoItems_models_classes_ItemCompiler extends tao_models_classes_Compiler
```

Comment:
are you sure that this class is not instantiated anywhere?

Reply:
yes